example/App.js

```diff
@@ -182,6 +184,18 @@ export default function App() {
     setCallMuted(callUUID, muted);
   };
 
+  const updateDisplay = (callUUID) => {
+    const number = calls[callUUID];
+    // Workaround because Android doesn't display well displayName, se we have to switch ...
+    if (isIOS) {
```

Comment:
@kylekurz @bhuangy @danjenkins on Android I have to switch `displayName` and `handle` arguments to have them displayed correctly (unless the `displayName` is not taken in account). 
Weirdly the Android code [seems to be OK](https://github.com/react-native-webrtc/react-native-callkeep/blob/master/android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java#L339-L340) ...

Have you this issue too ? I can reproduce with the example on a Pixel 1 (Android 9).

Reply:
I don't use the updateDisplay api yet so I can't comment :)

Reply:
I'd rather update the underlying modules so they can both have the arguments in the same order, instead of doing a platform check here, personally...

Reply:
Me too but my point is that the arguments on the Android side [are good](https://github.com/react-native-webrtc/react-native-callkeep/blob/master/android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java#L339-L340). I don't want to call `setAddress` with the displayName and `setCallerDisplayName` with the handle ...

Reply:
This doesn't really make sense. We should be figuring out what's going on underneath instead of providing a strange example.

For our app, we have the following scenarios:

1. Incoming call - Always displays name properly
2. Outgoing call started from our app - Always displays name properly
3. Outgoing call started from native dialer - Only displays the number

Does this match your experience? I think if you switch the items, you'll end up with a call log that has the right name, but the URI will be useless if someone tries to call it back.